src/panels/config/integrations/integration-panels/zwave_js/capability-controls/zwave_js-capability-control-door-lock.ts

```diff
@@ -0,0 +1,340 @@
+                )}
+              >
+                <ha-switch
+                  @change=${this._booleanChanged}
+                  key="blockToBlock"
+                  .checked=${this._configuration?.blockToBlock}
+                >
+                </ha-switch>
+              </ha-formfield>
+            </div>
+          `
+        : nothing}
+      ${this._capabilities?.autoRelockSupported
+        ? html`
+            <div class="row">
+              <ha-textfield
+                type="number"
+                .label=${this.hass.localize(
+                  "ui.panel.config.zwave_js.node_installer.capability_controls.door_lock.auto_relock_time"
+                )}
+                .value=${this._configuration?.autoRelockTime?.toString() ?? ""}
+                @change=${this._numberChanged}
+                key="autoRelockTime"
+              >
+              </ha-textfield>
+            </div>
+          `
+        : nothing}
+      ${this._capabilities?.holdAndReleaseSupported
+        ? html`
+            <div class="row">
+              <ha-textfield
+                type="number"
+                .label=${this.hass.localize(
+                  "ui.panel.config.zwave_js.node_installer.capability_controls.door_lock.hold_release_time"
+                )}
+                .value=${this._configuration?.holdAndReleaseTime?.toString() ??
+                ""}
+                @change=${this._numberChanged}
+                key="holdAndReleaseTime"
+              >
+              </ha-textfield>
+            </div>
+          `
+        : nothing}
+
+      <div class="actions">
+        <ha-progress-button
+          @click=${isValid ? this._saveConfig : undefined}
+          .disabled=${!isValid}
+        >
+          ${this.hass.localize("ui.common.save")}
+        </ha-progress-button>
+      </div>
+    `;
+  }
+
+  protected firstUpdated() {
+    this._loadConfiguration();
+    this._loadCapabilities();
+  }
+
+  private async _loadConfiguration() {
+    try {
+      const config = await invokeZWaveCCApi<DoorLockConfiguration | null>(
+        this.hass,
+        this.device.id,
+        this.command_class,
+        this.endpoint,
+        "getConfiguration",
+        [],
+        true
+      );
+      this._configuration = config ?? {
+        // The server can return null but I think a real device will always have a configuration
+        operationType: 1,
+        outsideHandlesCanOpenDoorConfiguration: [false, false, false, false],
+        insideHandlesCanOpenDoorConfiguration: [false, false, false, false],
+      };
+    } catch (err) {
+      this._error = extractApiErrorMessage(err);
+    }
+  }
+
+  private async _loadCapabilities() {
+    try {
+      const capabilities = await invokeZWaveCCApi<DoorLockCapabilities | null>(
+        this.hass,
+        this.device.id,
+        this.command_class,
+        this.endpoint,
+        "getCapabilities",
+        [],
+        true
+      );
```

Comment:
This call can throw with an error code `ZWaveErrorCodes.CC_NotSupported` if the `getCapabilities` command isn't supported by the device (i.e. if it's an older lock).

We need to that case and assume that autoRelock, holdAndRelease, twistAssist and blockToBlock are **not** supported.
The cleaner way would be to ask the driver for its cached information, but this isn't exposed yet: https://github.com/zwave-js/node-zwave-js/issues/7388

Reply:
Also in the error case, these door lock modes should be assumed to be supported:
![grafik](https://github.com/user-attachments/assets/a8a3e321-d665-499f-881e-e5c7a1f83792)

Reply:
Do we need to support get/set mode in the expert UI? I left it out because it's not part of getConfiguration/setConfiguration. I can add it with extra requests for the mode

Reply:
No, just setting is enough for certification. Getting wouldn't hurt, but you do that anyways on load if I'm not mistaken?

Reply:
Added a select for the mode with validation for the mode.
For `get` there is `currentMode` and `targetMode`. I used `currentMode` but please correct me if this is wrong.
It defaults to `Unsecured 0x00` when the driver doesn't return the mode on `get`. Maybe it should be `Unknown 0xfe` but this value is not in the above list so I set it to 0 for now.